.changelog/25998.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:security
+template: Adds sprig functions to the DefaultTemplateFunctionDenylist.
```

Comment:
We should also have a changelog entry for CT if we're picking up any CT CVEs here.